modules/ztp-gitops-ztp-max-spoke-clusters.adoc

```diff
@@ -0,0 +1,97 @@
+
+Network bandwidth and latency::
+Slow or high-latency network connections between the hub cluster and managed clusters can impact how the hub cluster manages multiple clusters.
+
+Managed cluster size and complexity::
+The size and complexity of the managed clusters also affects the capacity of the hub cluster.
+Larger managed clusters with more nodes, namespaces, and resources require additional processing and management overhead.
+Similarly, clusters with complex configurations or diverse workloads can require more resources from the hub cluster.
+
+Number of managed policies::
+The number of policies managed by the hub cluster scaled over the number of managed clusters bound to those policies is an important factor that determines how many clusters can be managed.
+
+Monitoring and management workloads::
+{rh-rhacm} continuously monitors and manages the managed clusters.
+The number and complexity of monitoring and management workloads running on the hub cluster can affect its capacity.
+Intensive monitoring or frequent reconciliation operations can require additional resources, potentially limiting the number of manageable clusters.
+
+{rh-rhacm} version and configuration::
+Different versions of {rh-rhacm} can have varying performance characteristics and resource requirements.
+Additionally, the configuration settings of {rh-rhacm}, such as the number of concurrent reconciliations or the frequency of health checks, can impact the managed cluster capacity of the hub cluster.
+
+Use the following representative configuration and network specfications to develop your own Hub cluster and network specifications.
+
+[IMPORTANT]
+====
+The following guidelines are based on internal benchmark testing only and do not represent a complete real-world host specification.
+====
+
+.Representative three-node hub cluster machine specifications
+[cols=2*, width="90%", options="header"]
+|====
+|Requirement
+|Description
+
+|{product-title} version
+|version 4.13
+
+|{rh-rhacm} version
+|version {rh-rhacm-version}
+
+|{cgu-operator-first}
+|version 4.13
+
+|Server hardware
+|3 x Dell PowerEdge R650 rack servers
+
+|NVMe hard disks
+a|* 50 GB disk for `/var/lib/etcd`
+* 2.9 TB disk for `/var/lib/containers`
+
+|SSD hard disks
+a|* 1 SSD split into 15 200GB thin-provisioned logical volumes provisioned as `PV` CRs
+* 1 SSD serving as an extra large `PV` resource
+|====
+
+[IMPORTANT]
+====
+The following network specifications are based on a simulated lab environment and represent best guess data points for a real-world RAN network.
+====
+
+.Simulated managed cluster network specifications
+[cols=2*, width="90%", options="header"]
+|====
+|Specification
+|Description
+
+|Round-trip time (RTT) latency
+|50ms RTT measured in the lab environment, representative of real-world network edge environment
+
+|Packet loss
+|0.02% packet loss in the lab environment
+
+|Lab bandwidth tests
+a|* 20 Mbps
+* 100 Mbps
+* Unlimited
```

Comment:
I don't understand this.
What is unlimited bandwidth?
And at which bandwidth was managing 3500 SNOs achieved?